design-tokens/tokens/layer1/base.json

```diff
@@ -1,109 +1,121 @@
       },
-      "lg": {
-        "value": "{size.dimension.lg}",
-        "type": "spacing"
+      "300": {
+        "value": "1.5rem",
+        "type": "dimension"
       },
-      "xl": {
-        "value": "{size.dimension.xl}",
-        "type": "spacing"
+      "400": {
+        "value": "2rem",
+        "type": "dimension"
       },
-      "multi-value": {
-        "value": "{size.dimension.sm} {size.dimension.xl}",
-        "type": "spacing",
-        "description": "You can have multiple values in a single spacing token. Read more on these: https://docs.tokens.studio/available-tokens/spacing-tokens#multi-value-spacing-tokens"
-      }
-    },
-    "borderRadius": {
-      "sm": {
-        "value": "4",
-        "type": "borderRadius"
+      "500": {
+        "value": "2.5rem",
+        "type": "dimension"
       },
-      "lg": {
-        "value": "8",
-        "type": "borderRadius"
+      "600": {
+        "value": "3rem",
+        "type": "dimension"
       },
-      "xl": {
-        "value": "16",
-        "type": "borderRadius"
+      "800": {
+        "value": "4rem",
+        "type": "dimension"
+      }
+    },
+    "sizes": {
+      "global": {
+        "xs": {
+          "value": "0.25rem",
+          "type": "dimension"
+        }
       },
-      "multi-value": {
-        "value": "{size.borderRadius.sm} {size.borderRadius.lg}",
-        "type": "borderRadius",
-        "description": "You can have multiple values in a single radius token. Read more on these: https://docs.tokens.studio/available-tokens/border-radius-tokens#single--multiple-values"
+      "icons": {
+        "sm": {
+          "value": "1rem",
+          "type": "dimension"
+        },
+        "md": {
+          "value": "1.25rem",
+          "type": "dimension"
+        },
+        "lg": {
+          "value": "1.5rem",
+          "type": "dimension"
+        }
       }
     }
   },
-  "opacity": {
-    "low": {
-      "value": "10%",
-      "type": "opacity"
+  "borderRadius": {
+    "sm": {
+      "value": "2",
+      "type": "borderRadius"
+    },
+    "lg": {
+      "value": "8",
+      "type": "borderRadius"
     },
     "md": {
-      "value": "50%",
-      "type": "opacity"
+      "value": "4",
+      "type": "borderRadius"
+    },
+    "none": {
+      "value": "0",
+      "type": "borderRadius"
     },
-    "high": {
-      "value": "90%",
-      "type": "opacity"
+    "circle": {
+      "value": "700",
+      "type": "borderRadius"
     }
   },
   "fontFamilies": {
-    "heading": {
-      "value": "Inter",
+    "mono": {
+      "value": "Ubuntu Mono",
       "type": "fontFamilies"
     },
-    "body": {
-      "value": "Roboto",
+    "sans": {
+      "value": "Inter",
       "type": "fontFamilies"
     }
   },
   "lineHeights": {
     "heading": {
-      "value": "110%",
+      "value": "132%",
       "type": "lineHeights"
     },
     "body": {
-      "value": "140%",
+      "value": "142%",
       "type": "lineHeights"
     }
```

Comment:
NP: Shouldn't these be in `rem`?

Reply:
Good point. I'm not sure.
The goal here is that these 2 values will be the multipliers that you can use to create the line height. So, if your font size is 1rem, the line height is going to be (1rem * 142% = 22,72). What value should it be in rem that allows us to reach the same effect?

Reply:
Ah interesting, that's a good point. Let's keep it in %